agents/agents-features/agents-features-opentelemetry/src/jvmTest/kotlin/ai/koog/agents/features/opentelemetry/event/ToolMessageEventTest.kt

```diff
@@ -20,7 +20,7 @@ class ToolMessageEventTest {
         val toolMessageEvent = ToolMessageEvent(
             provider = llmProvider,
             toolCallId = toolCallId,
-            toolResult = toolResult,
+            content = toolResult.toStringDefault(),
```

Comment:
This doesn’t relate to this PR, but I want to point out that `toStringDefault` raises for me some questions: why not just `toString`? what does `StringDefault` mean?

Reply:
Valid point! I think it is just a question about naming. If you look the implementation, it serialise the result into string. Probably, I would rename it in the other PR in future.